forms/models/element.js

```diff
@@ -247,12 +249,13 @@ define(function (require) {
       }
 
       result = Backbone.Model.prototype.set.call(this, attrs, options);
-
-      if ('value' in attrs) {
-        if (!options || !options.hasOwnProperty('value') || options.validate) {
-          this.isValid();
+      ['value', 'blob'].forEach(function (prop) {
```

Comment:
This is clever, and makes sense if there are going to be lots of properties that need special behaviour.

At the moment, however, we only have 2 properties ("value" and "blob"), and this `#set()` method is called many many times (e.g. dozens of times for each field). As such, I think extending the original `if`-condition is more appropriate than creating a new function (which is expensive) and looping (which is more expensive than a basic `if`).

There will be cases where the proposed form is more appropriate, especially because it's maintainable and quite legible, but we need to keep it as simple as possible here for performance reasons.

e.g.

```js
if ('value' in attrs || 'blob' in attrs) {
```